src/django/api/serializers.py

```diff
@@ -1042,12 +1043,25 @@ def get_sector(self, facility):
             .values('updated_at',
                     contributor_id=F('source__contributor_id'),
                     contributor_name=F('source__contributor__name'),
-                    values=F('sector'))
+                    values=F('sector'),
+                    is_from_claim=Value(False, models.BooleanField()))
+        claim_sectors = FacilityClaim \
+            .objects \
+            .filter(facility=facility, status=FacilityClaim.APPROVED) \
+            .order_by('contributor_id', '-updated_at') \
+            .distinct('contributor_id') \
```

Comment:
It looks like it isn't intended to be possible to have multiple approved `FacilityClaim` objects for the same `Facility`, but since there aren't any constraints enforcing that at the database-level I wanted to be safe here and assume it might be possible